util/cron/test-slurm-gasnet-mpi.bash

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+#
+# Test gasnet mpi against hellos on cray-cs.
+
+CWD=$(cd $(dirname $0) ; pwd)
+source $CWD/common-slurm-gasnet.bash
+
+export CHPL_NIGHTLY_TEST_CONFIG_NAME="gasnet-mpi"
```

Comment:
Think this has to be "slurm-gasnet-mpi" (I thought travis would complain about this, but it doesn't seem like it did)